Car_homework/src/com/company/Main.java

```diff
@@ -0,0 +1,34 @@
+package com.company;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+
+        VW car = new VW();
+        car.setNameModel("Jetta");
+        car.chooseCar(car.getNameModel());
+        car.setNameModel("CC");
+        car.chooseCar(car.getNameModel());
+        car.NameModelConstructor("Passat");
+        car.chooseCar(car.getNameModel());
+
+
+        BMW car2 = new BMW();
```

Comment:
You have too much hard code, better to have one class with all fields, and user enum for model